cli/azd/pkg/devcenter/manager.go

```diff
@@ -15,6 +17,9 @@ import (
 	"golang.org/x/exp/slices"
 )
 
+// ADE Bicep deployments have a name of a date like string followed by a number
+var bicepDeploymentNameRegex = regexp.MustCompile(`^\d{4}-\d{2}-\d{2}-\d+$`)
```

Comment:
Just to understand.. Is the regex also working for other provider deployments?

Reply:
This would only be used for ade/devcenter deployments.  We have a post-build TODO item to integrate with their new OUTPUTS api and at that point we will be able to apply a standard strategy for all ADE deployments.